src/DotNetty.Codecs.Redis/RedisBulkStringAggregator.cs

```diff
@@ -0,0 +1,219 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace DotNetty.Codecs.Redis
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Diagnostics.Contracts;
+    using DotNetty.Buffers;
+    using DotNetty.Codecs.Redis.Messages;
+    using DotNetty.Common.Utilities;
+    using DotNetty.Transport.Channels;
+
+    public sealed class RedisBulkStringAggregator : MessageToMessageDecoder<IRedisMessage>
```

Comment:
>RedisBulkStringAggregator [](start = 24, length = 25)

would it be hard to maintain parity with netty and have MessageAggregator as a base class, i.e. port MessageAggregator as well?